.github/workflows/test-chart.yaml

```diff
@@ -0,0 +1,41 @@
+# validate any chart changes under charts directory
+name: Chart Test
+
+env:
+  HELM_VERSION: v3.17.3
+
+on:
+  push:
+    # Exclude branches created by Dependabot to avoid triggering current workflow
+    # for PRs initiated by Dependabot.
+    branches-ignore:
+      - 'dependabot/**'
+  pull_request:
+    # paths:
+    #   - "charts/**"
+
+permissions:
+  contents: read
+
+jobs:
+  chart-lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: install Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v4
```

Comment:
Perhaps the preference is to complete the installation of helm in the Makefile. Here, it's just a quick start. I'd be more than happy to update. like add a command: `make helm` to install helm.